### PR TITLE
Create 2022-12-13-stop-trying-to-make-things-perfect.md

### PR DESCRIPTION
**Pull request by koddsson**

### CODE REVIEW

File: _posts/2022-12-13-stop-trying-to-make-things-perfect.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+Thinking about the cookbook I've gotten ahead of myself and thought about writing a intro to it. I think this is often my problem; over thinking. Realistically I'm making a cookbook so it's easy for me to whip up something good in the kitchen. It's not for anybody else. Tom even has a rule against this way of thinking; [Creativity is the enemy](https://ropac.net/exhibitions/547-tom-sachs-creativity-is-the-enemy/)
+
+I should just write some recipes down so it's easy for me to follow them and it's easy to add or change stuff. If I want an intro to it or change some recipes I should leave space for it to be added later. A few pages can be left at the beginning, a spacious margin for edits. Pages could even be glued in if needed!
+
+I should stop trying to make the thing perfect before I've even started. Just make the thing first!
```
Comment (koddsson) [inline _posts/2022-12-13-stop-trying-to-make-things-perfect.md:15]:
```suggestion
---
title: Stop trying to make things perfect
datetime: 2022-12-13T01:30
description: "A blog post with some thoughts on perfectionism"
---

I recently discovered the work of Tom Sachs, a contemporary artist known for his sculptural installations that often incorporate everyday objects and materials. After watching a number of his videos and reading about his creative process, I was struck by his philosophy of "just make the thing." In other words, don't try to overthink or perfect your work before you've even started - just get started and see where it takes you.

This philosophy resonated with me, as I often struggle with perfectionism in my own creative pursuits. For example, I have been thinking about creating a physical cookbook for some time now, but I have been stuck in the planning phase, trying to figure out the perfect layout and design before I even start writing any recipes. However, after learning about Tom Sachs' approach, I realized that I should just start writing down my recipes and worry about the design later. As Tom puts it, "Creativity is the enemy" - by trying to make everything perfect before we start, we risk stifling our own creativity and productivity.

To illustrate this point, consider the example of my mom's old cookbook. She used to collect newspaper clippings with recipes and handwritten notes, and would mark the good ones with "Great!" and add notes to the margins. This was not a fancy or professionally-designed cookbook, but it was functional and effective for her needs. It also allowed for flexibility and growth - she could easily add new recipes or make changes to existing ones by adding pages or making notes in the margins. This is the kind of cookbook I want to create for myself - one that is functional, flexible, and easy to use in the kitchen.

In conclusion, I think it's important to remember that perfectionism is often the enemy of creativity and productivity. Instead of trying to make everything perfect before we start, we should just get started and see where it takes us. By embracing the philosophy of "just make the thing," we can open ourselves up to new possibilities and creative growth.
```
